### PR TITLE
Fix hotspots issues in Android by using e.preventDefault() only for iPhone

### PR DESCRIPTION
**Pull request by Getitdan**
Hotspots work perfectly fine on Desktop & iPhone (Safari / Chrome). However, hotspots won't be responsive in Android at all.

Fixes issue #62.

### CODE REVIEW

**Comment by Getitdan** [comment]
I signed it!

File: src/Viewer.js [hunk 1]
```diff
@@ -136,9 +136,14 @@ function Viewer(domElement, opts) {
   setFullSize(this._controlContainer);
 
   // Prevent bounce scroll effect on iOS.
-  this._controlContainer.addEventListener('touchstart', function(event) {
-    event.preventDefault();
-  });
+  // Applied only for iPhone, as Android's events must have the default action to allow a proper hotspots' usage
+  var userAgent = navigator.userAgent || navigator.vendor || window.opera;
```
Comment (tjgq) [inline src/Viewer.js]: We use the `bowser` library for user agent detection. Please do `var browser = require('bowser')` at the top, then rewrite this as `if (browser.ios)`.

File: src/Viewer.js [hunk 2]
```diff
@@ -136,9 +136,14 @@ function Viewer(domElement, opts) {
   setFullSize(this._controlContainer);
 
   // Prevent bounce scroll effect on iOS.
-  this._controlContainer.addEventListener('touchstart', function(event) {
-    event.preventDefault();
-  });
+  // Applied only for iPhone, as Android's events must have the default action to allow a proper hotspots' usage
```
Comment (tjgq) [inline src/Viewer.js]: ... "to allow interaction with hotspots."

Comment (Getitdan) [inline src/Viewer.js]: Changed

File: src/Viewer.js (hunk 1 above)
Comment (Getitdan) [inline src/Viewer.js]: Thanks, done!

**Comment by tjgq** [comment]
Sorry - it looks like your commits aren't associated with the email address on your Github profile (the author shows up as "dummyuser" in the Commit tab). Our CLA policy requires that they match.

Can you please amend the author of your commits, make sure you've signed the CLA with the same email, and reopen the pull request? Thanks.

**Comment by Getitdan** [comment]
Hey tjgq, thank you a lot. Of course I will.

**Pull request merged**